client/my-sites/post-share/index.jsx

```diff
@@ -0,0 +1,491 @@
+			userCurrency,
+			sharingActions,
+		} = this.props;
+
+		if ( planSlug !== PLAN_BUSINESS ) {
+			return (
+				<Banner
+					className="post-share__footer-banner"
+					callToAction={
+						translate( 'Upgrade for %s', {
+							args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+							comment: '%s will be replaced by a formatted price, i.e $9.99'
+						} )
+					}
+					description={ translate( 'Live chat support and no advertising.' ) }
+					list={ [
+						translate( 'Live chat support' ),
+						translate( 'No Advertising' )
+					] }
+					plan={ PLAN_BUSINESS }
+					title={ translate( 'Upgrade to a Business Plan!' ) }
+				/>
+			);
+		}
+
+		return sharingActions.map(
+			( item, index ) => this.renderFooterSectionItem( item, index )
+		);
+	}
+
+	renderPublishedList() {
+		return 'published';
+	}
+
+	renderFooter() {
+		const { footerSection } = this.state;
+
+		return (
+			<div className="post-share__footer">
+				<SectionNav className="post-share__footer-nav" selectedText={ 'some text' }>
+					<NavTabs label="Status" selectedText="Published">
+						<NavItem
+							selected={ footerSection === PostShare.FOOTER_SECTION_SCHEDULED }
+							count={ 4 }
+							onClick={ this.setFooterSection( PostShare.FOOTER_SECTION_SCHEDULED ) }
+						>
+							Scheduled
+						</NavItem>
+						<NavItem
+							selected={ footerSection === PostShare.FOOTER_SECTION_PUBLISHED }
+							count={ 2 }
+							onClick={ this.setFooterSection( PostShare.FOOTER_SECTION_PUBLISHED ) }
+						>
+							Published
+						</NavItem>
+					</NavTabs>
+				</SectionNav>
+				<div className="post-share__scheduled-list">
+					{ footerSection === PostShare.FOOTER_SECTION_SCHEDULED &&
+						this.renderScheduledList()
+					}
+				</div>
+				<div className="post-share__published-list">
+					{ footerSection === PostShare.FOOTER_SECTION_PUBLISHED &&
+						this.renderPublishedList()
+					}
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		if ( ! this.props.isPublicizeEnabled ) {
+			return null;
+		}
+
+		if ( ! this.props.planHasRepublicizeFeature ) {
+			return ( <div className="post-share">
+				<Banner
+					feature="republicize"
+					title={ this.props.translate( 'Unlock the ability to re-share posts to social media' ) }
+					callToAction={ this.props.translate( 'Upgrade to Premium' ) }
+					description={ this.props.translate( 'Get unlimited premium themes, video uploads, monetize your site and more.' ) }
+				/>
+			</div> );
+		}
+
+		const classes = classNames( 'post-share__wrapper', {
+			'has-connections': this.hasConnections()
+		} );
+
+		const {
+			connections,
+			failure,
+			postId,
+			requesting,
+			siteId,
+			siteSlug,
+			success,
+			translate,
+		} = this.props;
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
+		return (
+			<div className="post-share">
+				{ requesting &&
+					<Notice status="is-warning" showDismiss={ false }>
+							{ translate( 'Sharing…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Sharing...' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/publicize-scheduling)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).